test-requirements.txt

```diff
@@ -1,5 +1,5 @@
 ddt==0.8.0
-django-nose==1.4.1
+django-nose==1.4.4
```

Comment:
Could you update the tox requirement in here to the latest version (2.7.0) as well? I was having some issues with that in my ora work.